### ISSUE TITLE
why did you set Normalize(mean=[0.6071, 0.4609, 0.3944], std=[0.2457, 0.2175, 0.2129])

### ISSUE THREAD

**Issue by eeric** [tamerthamoqa/facenet-pytorch-glint360k#16]
usually: Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
insightface: Normalize(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5])
the reason why your set?

__

**Comment by tamerthamoqa** [comment]
Hello @eeric,

I set the normalization parameters as per the calculated training dataset RGB mean and standard deviation values in this [script](https://github.com/tamerthamoqa/facenet-pytorch-glint360k/blob/master/datasets/calculate_glint360k_rgb_mean_std.py) . It may or not be a better normalization technique than the one used by insightface and other face recognition repositories, but I have chosen using the training dataset values as good practice.

**Comment by eeric** [comment]
thanks!
